ProcessMaker/Repositories/CaseRepository.php

```diff
@@ -69,34 +68,27 @@ public function create(ExecutionInstanceInterface $instance): void
      */
     public function update(ExecutionInstanceInterface $instance, TokenInterface $token): void
     {
+        if (is_null($instance->case_number)) {
+            \Log::error('Case number is required. instance: ' . $instance->getKey());
```

Comment:
How this scenario could happen, could you add some steps or a test to replicate it?
Maybe the system processes like action by email, or data connectors with callback

Reply:
This scenario was given in this test `UploadPublicFileTest`, since the `processRequest` is instantiated but not stored